### DISCUSSION TITLE
question: new windows forms task dialogs

### DISCUSSION THREAD

**Discussion by ghost** [dotnet/winforms#4291] [Q&A] [answered]
hi

I have read windows forms in .net 5.0 has a new task dialog window. It was contributed by an open source developer.
However I can't find this anywhere and also no docs.

Can somebody explain me how to create this in a windows forms .net 5.0 program?

Jessica

**Comment by paul1956** [comment, marked as answer]
https://docs.microsoft.com/en-us/samples/dotnet/samples/winforms-taskdialog-vb/ or https://docs.microsoft.com/en-us/samples/dotnet/samples/winforms-taskdialog-cs/ you can then browse the many samples. There is no designer support for this that I can find.

**Reply by RussKie** [reply]
Thank you @paul1956 
> There is no designer support for this that I can find.

There is currently no designer support for Task Dialog.

@ancientmagicwizard the docs are here: https://docs.microsoft.com/dotnet/api/system.windows.forms.taskdialog
